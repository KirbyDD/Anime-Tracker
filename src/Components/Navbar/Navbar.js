import React, { Component } from 'react';
import { NavLink } from 'react-router-dom'
import './Navbar.scss';

class Navbar extends Component {
  constructor() {
    super();
    this.state = {
      selectedTab: 'Home'
    }
  }

  render() {
    return (
      <nav className="navbar">
        <NavLink to='/' className="tab" exact activeClassName="selected">Home</NavLink>
        <NavLink to='/forums' className="tab" exact activeClassName="selected">Forums</NavLink>
        <NavLink to='/about' className="tab" exact activeClassName="selected">About</NavLink>
      </nav>
    )
  }
}

export default Navbar;