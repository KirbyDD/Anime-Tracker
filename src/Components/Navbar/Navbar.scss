.navbar {
  display: flex;
  justify-content: space-around;
  margin: 35px;
}

.tab {
  background-color: black;
  box-shadow: 3px 5px 3px gold;
  color: gold;
  font-size: 33px;
  height: 70px;
  width: 315px;
  text-align: center;
  text-decoration: none;
  &:hover {
    box-shadow: 3px 5px 3px white;
    color: white;
    transition: .3s;
  }
}

.selected {
  background-color: gold;
  color: black;
  box-shadow: 3px 5px 3px white;
}

@media only screen and (max-width: 600px) {
  .tab {
    font-size: 18px;
    width: 90px;
  }

  .navbar {
    margin: 5px;
  }
}
