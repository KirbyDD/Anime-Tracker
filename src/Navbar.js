import React, { Component } from 'react'

class Navbar extends Component {
  constructor() {
    super();
    this.state = {
      selectedTab: 'Home'
    }
  }

  render() {
    return (
      null
    )
  }
}

export default Navbar;