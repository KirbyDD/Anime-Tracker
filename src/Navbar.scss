.navbar {
  display: flex;
  justify-content: space-around;
  margin: 35px;
}

.tab {
  background-color:black;
  box-shadow: 3px 5px 3px gold;
  color: gold;
  font-size: 1rem;
  height: 40px;
  width: 115px;
}