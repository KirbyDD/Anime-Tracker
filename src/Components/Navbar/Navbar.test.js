import React from 'react';
import {shallow} from 'enzyme';
import {Navbar} from './Navbar';

describe('Navbar', () => {
  let wrapper;

  beforeEach(function() {
    wrapper = shallow(
      <Navbar />
    )
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})