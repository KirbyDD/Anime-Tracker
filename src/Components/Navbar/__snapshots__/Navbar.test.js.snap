// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<nav
  className="navbar"
>
  <NavLink
    activeClassName="selected"
    className="tab"
    exact={true}
    to="/"
  >
    Home Page
  </NavLink>
  <NavLink
    activeClassName="selected"
    className="tab"
    exact={true}
    to="/watchlist"
  >
    Your Watchlist
  </NavLink>
</nav>
`;
